packages/arb-token-bridge-ui/src/components/MainContent/MainContent.tsx

```diff
@@ -130,8 +123,8 @@ export function MainContent() {
         {/* Transaction history - pending transactions + history table */}
         <TransactionHistory
           {...{
-            depositsPageParams,
-            withdrawalsPageParams,
+            depositsPageParams: { ...depositsPageParams, pageSize },
+            withdrawalsPageParams: { ...withdrawalsPageParams, pageSize },
```

Comment:
`TransactionHistory` has other issues, but that would return new references every time and cause a new render, no?

Reply:
I haven't experienced anything like that, though we could memoize the entire object with the updated `pageSize`, to be safe